packages/rich-text/src/formatters/test/letterSpacing.tsx

```diff
@@ -35,16 +43,16 @@ const { elementToStyle, stylesToCSS, getters, setters } = formatter;
 
 describe('Color formatter', () => {
   beforeAll(() => {
-    getPrefixStylesInSelection.mockImplementation(() => [NONE]);
+    jest.mocked(getPrefixStylesInSelection).mockImplementation(() => [NONE]);
```

Comment:
I didn't know about [this one](https://jestjs.io/docs/mock-function-api/#jestmockedsource-options). Nice!